README.md

```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
 ## Team:
 
 Developers: @Amirk390 and @SamiSha99
+
 Scrum: @Karyum
+
 QA: @matthewdking
 
 ## Tech stack:
 
 Frontend framework: React.
-Backend: none.
+
 CSS: Styled-components.
+
 Compiler: Webpack with Babel.
+
 Linting: ESLint with React config.
 
 
 ## Local instructions
 
 
 1. clone it `git clone git@github.com:FACN3/washinator.git` in your tarminal
-+ then `npm i`
+2. install npm packages  `npm i`
 
 See also the list of [contributors](https://github.com/FACN3/washinator/graphs/contributors) who participated in this project.
 
-## Our ProtoType
+# Our ProtoType
+Below, we have some user storys and the solutions for that storys
 
-[Our InVision ProtoType](https://invis.io/UNFO2C1YA2P)
 
-[Download link](https://projects.invisionapp.com/d/zipexport/download/projectid/13267105)
+### User Story:
+:man: As a water consumer, I want, to calculate my overall water consumption.
+So that i can reduce my water expenses
 
 ### Calculator
 
 ![Calculator](https://user-images.githubusercontent.com/24490876/35803983-846a0468-0a7e-11e8-9b37-d00265775367.png)
 
+### User Story:
+:raising_hand: As a Water consumer, I want to choose a certain activity and know how much water being used also how much money dollar So that I would be more careful with using Water.
```

Comment:
As a water consumer, I want to choose certain activities and know how much water I am using and how much money this equates to, so that I will be more careful with the amount of water I use.